torch/_custom_op/impl.py

```diff
@@ -811,7 +812,14 @@ def error_fn(what):
             if not schema_type.startswith("Tensor"):
                 error_fn(f"Parameter {name} is in mutable_args but only Tensors or collections of Tensors can be mutated")
             schema_type = f"Tensor(a{idx}!){schema_type[len('Tensor'):]}"
+        seen_args.add(name)
         params.append(f"{schema_type} {name}")
+    mutated_args_not_seen = set(mutated_args) - seen_args
```

Comment:
Shouldn't mutated_args always be a set?

Reply:
It's an Iterable (list, tuple, set, etc). I didn't want to make it set all the time because the empty set is kinda weird: set()